### PR TITLE
Add static UI to WeConnect - a web application

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

Add a static user interface to a web application

#### Description of Task to be completed?

Simulate the functionality of a website platform creates awareness for businesses.
Give users the ability to write reviews about the businesses they have interacted with. 
 
#### How should this be manually tested?

Visit the website: https://artorious.github.io/
Download or Clone: https://github.com/artorious/WeConnect.git

Navigate the website to ensure logical flow through the pages and links View the website on different sized screens to ensure responsiveness

A user should be able to log in, register a business account/ profile or prompted to sign up first if not registered. A signed in user should be able to Update a business profile

#### What are the relevant pivotal tracker stories?

[#155504557](https://www.pivotaltracker.com/story/show/155504557)
[#155504485](https://www.pivotaltracker.com/story/show/155504485)
[#155504574](https://www.pivotaltracker.com/story/show/155504574)

#### Any background context you want to add?

Target or expected users include business owners and their customers

#### Screenshots
screenshot from 2018-02-28 19 26 21

screenshot from 2018-02-28 19 27 30

screenshot from 2018-02-28 19 30 08

screenshot from 2018-02-28 19 29 13

screenshot from 2018-02-28 19 28 51

screenshot from 2018-02-28 19 28 09

### CODE REVIEW

**Comment by jm-wanja** [comment]
Hey @artorious you should add a description to your PR so that others know what is happening.

File: designs/UI/all_reviews.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>WeConnect</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" 
+    integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js"
```
Comment (cosmas28) [inline designs/UI/all_reviews.html]: What do you think about placing the Js files just before closing </body>

Comment (artorious) [reply designs/UI/all_reviews.html]: Good catch thanks. I remember reading somewhere that pages render better/faster if scripts are placed at the bottom of the page before closing. Is this the reason you for your recommend?

Comment (artorious) [reply designs/UI/all_reviews.html]: @jm-wanja Done. That was a big ommission on my part.my part. Thanks

Comment (artorious) [reply designs/UI/all_reviews.html]: @cosmas28 
Good catch thanks. I remember reading somewhere that pages render better/faster if scripts are placed at the bottom of the page before closing. Is this the reason you for your recommend

**Comment by jm-wanja** [comment]
@artorious on the *What are the relevant pivotal tracker stories* section you may include the link like so [insert-the-story id]     (copy-paste-story-link)

**Pull request merged**